atomic_reactor/plugins/pre_add_labels_in_df.py

```diff
@@ -196,6 +200,19 @@ def add_as_an_alias(new, old, is_old_inherited):
             self.log.debug("applied only some aliases, following old labels were not found: %s",
                            ", ".join(not_applied))
 
+    def add_info_url(self, base_labels, df_labels, plugin_labels):
+        all_labels = base_labels.copy()
+        all_labels.update(df_labels)
+        all_labels.update(plugin_labels)
+
+        class MyFormatter(string.Formatter):
+            def get_field(self, field_name, args, kwargs):
+                return (self.get_value(field_name, args, kwargs), field_name)
```

Comment:
Could add a comment (maybe a class docstring on that class?) saying why we need to do this rather than just calling str.format().